cpuinfo/cpuinfo.py

```diff
@@ -600,16 +602,39 @@ def _asm_func(self, restype=None, argtypes=(), byte_code=[]):
 		if DataSource.is_windows:
 			# Allocate a memory segment the size of the byte code, and make it executable
 			size = len(byte_code)
+			# Alloc at least 1 page to ensure we own all pages that we want to change protection on
+			if size < 0x1000: size = 0x1000
```

Comment:
Any allocations over a page size are guaranteed to be allocations of entire pages.  Since we are marking these pages executable, this is important for security.